content/en/opentelemetry/otlp_metrics.md

```diff
@@ -0,0 +1,284 @@
+    text: "OpenTelemetry Environment Variable Spec"
+  - link: "https://opentelemetry.io/docs/specification/protocol/exporter/"
+    tag: "External Site"
+    text: "OpenTelemetry Protocol Exporter"
+  - link: "https://opentelemetry.io/docs/specs/otel/metrics/sdk_exporters/otlp/ "
+    tag: "External Site"
+    text: "OTLP Metrics Exporter"
+  - link: "/opentelemetry/guide/otlp_delta_temporality/"
+    tag: "Documentation"
+    text: "Configure delta temporality in OpenTelemetry"
+  - link: "/metrics/otlp/?tab=summary#mapping"
+    tag: "Documentation"
+    text: "OTLP Metrics Mapping in Datadog"
+---
+
+{{< callout header="false" btn_hidden="true">}}
+  The Datadog OTLP metrics intake endpoint is in private beta.
+{{< /callout >}} 
+
+{{< site-region region="ap1,gov" >}}
+<div class="alert alert-warning">Datadog OTLP metrics intake endpoint is not supported for your selected <a href="/getting_started/site">Datadog site</a> ({{< region-param key="dd_site_name" >}}).</div>
+{{< /site-region >}}
+
+## Overview
+
+Datadog's OpenTelemetry Protocol (OTLP) metrics intake API endpoint allows you to send metrics directly to Datadog. With this feature, you don't need to run the [Datadog Agent][2] or [OpenTelemetry Collector + Datadog Exporter][1].
+
+You might prefer this option if you're looking for a straightforward setup and want to send metrics directly to Datadog without using the Datadog Agent or OpenTelemetry Collector.
+
+This endpoint is particularly useful in the following scenarios:
+
+- **OpenTelemetry distributions without Datadog Exporter support**: Some OpenTelemetry distributions, such as the [AWS Distro for OpenTelemetry (ADOT)][8], have removed vendor-specific exporters in favor of a unified OTLP exporter. The OTLP metrics endpoint enables these distributions to send metrics directly to Datadog seamlessly.
+
+- **Technical constraints using the Datadog Exporter or Agent**: Ideal for scenarios where installing additional software is impractical or restrictive, such as third-party managed services (for example, Vercel), applications on customer devices, or environments requiring streamlined, Agentless observability pipelines. The OTLP metrics endpoint enables direct OTLP metric ingestion in these scenarios.
+
+## Configuration
+
+To export OTLP metrics data to the Datadog OTLP metrics intake endpoint:
+
+1. [Ensure only delta metrics are sent](#ensure-delta-temporality).
+1. [Configure the OTLP HTTP exporter](#configure-the-exporter).
+   - Set the Datadog OTLP metrics intake endpoint.
+   - Configure the required HTTP headers.
+1. (Optional) [Set the `dd-otel-metric-config` HTTP header](#optional-configure-the-metric-translator) to configure the metric translator behavior.
+
+### Configure the exporter
+
+To send OTLP data to the Datadog OTLP metrics intake endpoint, you need to use the OTLP HTTP Protobuf exporter. The process differs depending on whether you are using automatic or manual instrumentation for OpenTelemetry.
```

Comment:
In case of metrics HTTP JSON is also supported, but HTTP Protobuf is preferred because of performance.